test/functional/fixtures/upload/test.js

```diff
@@ -0,0 +1,42 @@
+var expect = require('chai').expect;
+
+
+describe('Upload', function () {
+    it('Should pass when uploading a file', function () {
+        return runTests('testcafe-fixtures/upload.test.js', 'Upload a file');
+    });
+
+    it('Should pass when uploading multiple files', function () {
+        return runTests('testcafe-fixtures/upload.test.js', 'Upload multiple files');
+    });

```

Comment:
These two tests above are not error-related, so their name shouldn't start with `Should pass` or `Should fail`. It should be something like `Should upload a file`